stable/aws-load-balancer-controller/templates/webhook.yaml

```diff
@@ -56,7 +56,7 @@ webhooks:
     resources:
     - pods
   sideEffects: None
-{{- if .Values.enableServiceMutatorWebhook }}
+{{- if or .Values.enableServiceMutatorWebhook .Values.serviceMutatorWebhook.enabled }}
```

Comment:
Should this be `and`'ed for backwards compatibility purposes? Or should `.Values.serviceMutatorWebhook.enabled` be set to `false`?

If someone sets:

```yaml
enableServiceMutatorWebhook: false
```

And the default is

```yaml
serviceMutatorWebhook:
  enabled: true
```

Then on upgrade this will install the `serviceMutatorWebhook` that they had disabled.

Reply:
Oh yeah, good call!